tool/tsh/tsh.go

```diff
@@ -3326,7 +3326,7 @@ func makeClientForProxy(cf *CLIConf, proxy string, useProfileLogin bool) (*clien
 	if cf.AuthConnector == constants.HeadlessConnector {
 		// Lock the process memory to prevent rsa keys and certificates from being exposed in a swap.
 		if err := mlock.LockMemory(); err != nil {
-			return nil, trace.Wrap(err, "failed to lock system memory for headless login")
+			fmt.Printf("WARNING: Failed to lock system memory for headless login: %v\n", err)
```

Comment:
What should the user do in this situation? Can we either recommend a fix inline or link to the docs on how to fix this?

Reply:
I like how Vault does it: https://support.hashicorp.com/hc/en-us/articles/115012787688-Vault-and-mlock-

```
$ vault server -config=example.hcl
Error initializing core: Failed to lock memory: cannot allocate memory

This usually means that the mlock syscall is not available.
Vault uses mlock to prevent memory from being swapped to
disk. This requires root privileges as well as a machine
that supports mlock. Please enable mlock on your system or
disable Vault from using it. To disable Vault from using it,
set the `disable_mlock` configuration option in your configuration
file.
```

Reply:
I like this error message it's really good. Explains what went wrong and what the user can do to fix it. :+1:

Reply:
I added a similar error message with a link to the docs.